### ISSUE TITLE
Typo in README

### ISSUE THREAD

**Issue by kpie89** [ga-wdi-boston/ruby-vs-js-arrays#6]
```
> not_the_same_type = [[], 'one', 2.0, 3]
=> [[], "one", 2.0, 3]
> developers = ['Ken', 'Keven', 'Khanhnhat', 'Kyle']
=> ["Ken", "Keven", "Khanhnhat", "Kyle"]
```

Please replace all instances of "Keven" with 'Kevin"

**Comment by jrhorn424** [comment]
@kpie89 Could you please make a Pull Request with your proposed changes? Thanks, "Kevin with an 'i'".

**Comment by kpie89** [comment]
Done and done. Thanks.

**Comment by jrhorn424** [comment]
Closed by #8
